File: app/src/main/java/com/tomaszrykala/dorefindmi/things/supplier/abcleds/AbcLedsSupplier.java
package com.tomaszrykala.dorefindmi.things.supplier.abcleds;

import com.tomaszrykala.dorefindmi.model.AbcLed;
import com.tomaszrykala.dorefindmi.things.supplier.BaseSupplier;

public interface AbcLedsSupplier extends BaseSupplier {

    void lightFor(boolean onPadA, boolean onPadB, boolean onPadC);

    boolean isLitAt(AbcLed abcLed);
}

File: app/src/main/java/com/tomaszrykala/dorefindmi/things/supplier/abcleds/AbcLedsSupplier.kt
package com.tomaszrykala.dorefindmi.things.supplier.abcleds

import com.tomaszrykala.dorefindmi.things.supplier.BaseSupplier

interface AbcLedsSupplier : BaseSupplier {

    fun lightFor(onPadA: Boolean, onPadB: Boolean, onPadC: Boolean)
}
